technical-reports/format/design-token.md

```diff
@@ -113,7 +125,7 @@ The `$type` property can be set on different levels:
 - at the group level
 - at the token level
 
-The `$type` property MUST be a plain JSON string, whose value is `string`, `number`, `boolean`, `object`, `array`, `null` or one of the values specified in respective [type chapters](#types). The value of `$type` is case-sensitive.
+The `$type` property MUST be a plain JSON string, whose value is one of the values specified in respective [type chapters](#types). The value of `$type` is case-sensitive.
```

Comment:
* I'd recommend removing the link to `#types` ("8 Types"), purely because local context would require linking to BOTH "8 Types" and "9 Composite types".  
* Additionally, I'd reword it as `... specified in respective type definitions.`

With the above changes, if § 8 and 9 were to be merged/reorganized in the future, then this content would still be correct and we could potentially reintroduce the link later.

```markdown
...values specified in respective type definitions. The value ...
```

Reply:
Good spot! I'll change it along the lines you've suggested.

Longer term, I think quite a few spec chapters could probably do with some re-organising. But that's for another day... ;-)